masterlist.yaml

```
@@ -5956,7 +5956,7 @@ plugins:
       - Scripts
 
   - name: 'New Vegas Redesigned 3.esm'
-    url: [ 'https://www.nexusmods.com/newvegas/mods/56312/' ]
+    url: [ 'https://www.nexusmods.com/newvegas/mods/72951' ]
```

We can keep the link for the old version if it's still available.
Trailing slashes `/` should be added to links as well, they're useful if you want to change multiple links quickly.

```suggestion
    url:
      - link: 'https://www.nexusmods.com/newvegas/mods/56312/'
        name: 'Fallout New Vegas Redesigned 3 - Open Source'
      - link: 'https://www.nexusmods.com/newvegas/mods/72951/'
        name: 'NVR3R - New Vegas Redesigned 3 Revised'
```

---

Sounds good! Yeah, Open Source is indeed the original.